src/mote/modules/call.py

```diff
@@ -26,16 +26,15 @@
 from datetime import datetime
 
 import bs4 as btsp
-
-recognition_pattern = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
+from flask import current_app as app
 
 
 def fetch_channel_dict():
     try:
         channel_dict = {}
-        chanlist = os.listdir("/srv/web/meetbot")
+        chanlist = os.listdir(app.config["MEETING_DIR"])
         for channel in chanlist:
-            channel_dict[channel] = "https://meetbot-raw.fedoraproject.org/%s" % channel
+            channel_dict[channel] = app.config["MEETBOT_URL"] + "/%s" % channel
```

Comment:
Instead of combining string concatenation & (old) %-format, you should use f-string everywhere. It improve readability, and is actually a bit faster. 
Ex.:
```suggestion
            channel_dict[channel] = f"{app.config['MEETBOT_URL']}/{channel}"
```